indexer/services/roundtable/src/config.ts

```diff
@@ -177,7 +177,8 @@ export const configSchema = {
   MAX_COMPLIANCE_DATA_AGE_SECONDS: parseInteger({ default: 2_630_000 }), // 1 month
   MAX_ACTIVE_COMPLIANCE_DATA_AGE_SECONDS: parseInteger({ default: 86_400 }), // 1 day
   MAX_COMPLIANCE_DATA_QUERY_PER_LOOP: parseInteger({ default: 100 }),
-  COMPLIANCE_PROVIDER_QUERY_BATCH_SIZE: parseInteger({ default: 100 }),
+  // v2/wallet/synchronous rate limit is 15/s https://developers.elliptic.co/docs/configuration
+  COMPLIANCE_PROVIDER_QUERY_BATCH_SIZE: parseInteger({ default: 15 }),
```

Comment:
Will this be enough to ensure all the addresses in the table are checked? Should the frequency of the loop be increased if not?

Reply:
The total number of addresses attempted per roundtable runs is unaffected. Currently ~60 requests are 200 because we still have ~40 invalid accounts. At the rate of ~60 per 5min, we should be catching up by ~15k a day, which is what i have observed. 
If the number of active accounts skyrockets or the number of invalid addresses increases to near 100, we will start to accumulate stale rows. The new metrics (will add alarming soon) will catch if this happens.